header-footer-grid/Core/Customizer/Description_Control.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Name:    Header Footer Grid - Description Control
```

Comment:
we already have this type of control somewhere, @abaicus used for blog typography shortcut

Reply:
I've removed this control, and modified `Responsive_Toggle` control from `React` to allow for description text not just links. Previously it was only allowing links to be passed.

 @abaicus If you have any input on this change let me know:  https://github.com/Codeinwp/neve/pull/2783/commits/40422d0215cbbd06f922eb2a1a17e19b5f0c7582

Reply:
@preda-bogdan we have already a non-responsive toggle used widely in theme